src/py_sonar_scanner/logger.py

```diff
@@ -37,3 +38,11 @@ def _setup_logger(log: Logger):
         log.setLevel(logging.INFO)
         handler = logging.StreamHandler()
         log.addHandler(handler)
+
+    @classmethod
+    def set_debug(cls, debug: bool) -> None:
+        if debug:
+            cls.get_logger().setLevel(logging.DEBUG)
```

Comment:
Following the other issue we had, I have a doubt. Should we set logging level explicitly in the other branch as well?
Do we have tests that assert which log level is set on the application logger? Maybe we need those as well to avoid bad surprises.